ios/Framework/Core/Base/BaseListScreenlet/BaseListView.swift

```diff
@@ -91,4 +93,23 @@ public class BaseListView: BaseScreenletView {
 		
 		return rows[key]!
 	}
+	
+	internal func areThereMoreRows(newRows: [String : [AnyObject]]) -> Bool {
+		if newRows.count == 0 {
+			return false
+		}
+		
+		let newRowsCount = newRows.values.reduce(0) { $0 + $1.count }
+		let isFirstPage = rows.values.reduce(0) { $0 + $1.count } == 0
```

Comment:
We can only know if we are in the first page verifying that there are no rows in any section